src/main/resources/db/migration/V12_Totrinnskontroll.sql

```diff
@@ -0,0 +1,16 @@
+CREATE table totrinnsstatus
+(
+    id             UUID PRIMARY KEY,
+    behandlings_id UUID NOT NULL REFERENCES behandling (id),
+    opprettet_av   VARCHAR NOT NULL,
+    opprettet_tid  TIMESTAMP(3) NOT NULL,
+    endret_av      VARCHAR NOT NULL,
+    sporbar        TIMESTAMP(3) NOT NULL,
```

Comment:
Hva er sporbar her? 
Er det kun årsak som kun burde være JSON? Mens de andre som er JSON burde være VARCHAR? 

Status burde være `NOT NULL`? 

Burde saksbehandler være `NOT NULL` ?

Reply:
sporbar er tanken at denne endres kvar gong det blir gjort noko med objektet. Og ja med ettertanken sp vil aldri saksbehandler vere null og status vil alltid vere satt.

Reply:
Hvis vi skal gjenbruke "sporbar" for dette objekt, så burde den følge liknende navn som de andre tabellene som finnes, `endretAv`. 
Då vi allerede har `saksbehandler` og `beslutter` her, så er jeg usikker på behovet for `Sporbar-objektet`. Men bare erstatter de 4 feltene koblet til sporbar til `opprettet_tid` og `besluttet_tid`, hva tenker @eirarset  ?
Husk formattering :)

Reply:
endrer sporbar til endret_tid, slik at den følger det som dei andre tabellene bruker. Behovet for sporbar-objektet ser eg at blir fort overengineering

Reply:
`opprettet_tid` og `endret_tid` høres bra ut